src/content/migrate/5.md

```diff
@@ -124,25 +137,55 @@ Using `/* webpackChunkName: '...' */`: Make sure to understand the intention:
 
-- When using `const compiler = webpack(...);`, make sure to close the compiler after using it: `compiler.close();`.
+- When using `const compiler = webpack(...);`, make sure to close the compiler after using it: `compiler.close(callback);`.
+    - This doesn't apply to the `webpack(..., callback)` form. It will automatically call close.
+    - This is optional if you use webpack in watching mode until the user ends the process. The idle phases in watch mode will be used for this kind of work.
 
 #### Run a single build and follow advises
 
+Please make sure to read errors/warnings carefully.
+
 If there is no corresponding advise? Please create an issue and we will try to resolve it. Repeat this step until you solved at least level 3 or 4:
 
-- Level 1: Schema validation fails. Configuration options have changed. There should be a validation error with a `BREAKING CHANGE:` note.
+- Level 1: Schema validation fails. Configuration options have changed. There should be a validation error with a `BREAKING CHANGE:` note, or a hint which option should be used instead.
 - Level 2: webpack exits with an error. The error message should tell you what needs to be changed.
 - Level 3: Build Errors. The error message should have a `BREAKING CHANGE:` note.
 - Level 4: Build Warnings. The warning message should tell you what can be improved.
-
-Deprecation warnings. You might get a lot of deprecation warnings. This is not a problem right now. Plugins need time to catch up with core changes. Please ignore them until release candidate out of Beta.
-
-- You can hide deprecation warnings by running node with `--no-deprecation` flag, e.g.: `node --no-deprecation node_modules/webpack/bin/webpack.js`. This should only be a temporary workaround.
-- Plugins and Loaders contributors can follow the advises in the deprecation messages to improve the code.
+- Level 5: Runtime Errors. This is tricky. You probably have to debug to find the problem. A general advise is difficult here.
+- Level 6: Deprecation Warnings. You probably get a lot of deprecation warnings. This is not directly a problem. Plugins need time to catch up with core changes. Please report these deprecations to the plugins. These deprecations are only warnings and the build will still work with only minor drawbacks (like less performance).
+- Level 7: Performance issues. Usually performance should improve with webpack 5, but there are also a few cases where performance get worse.
+
+- Regarding Runtime Errors:
+    - `process` is not defined.
+        - Webpack 5 do no longer include a polyfill for this Node.js variable. Avoid using it in frontend code.
+        - Want to support frontend and browser usage? Use the `exports` or `imports` package.json field to use different code depending on environment.
+            - To support older bundlers use the `browser` field too.
+            - Alternative: Wrap code blocks with `typeof process` checks. Note that this is worse regarding bundle size.
```

Comment:
```suggestion
            - Alternative: Wrap code blocks with the `typeof process` checks. Note that this will have a negative impact on the bundle size.
```